projects/plugins/jetpack/extensions/blocks/contact-form/variations.js

```diff
@@ -32,6 +32,18 @@ const variations = compact( [
 				},
 			],
 		],
+		attributes: {
```

Comment:
Shouldn't we avoid this code duplication? 
A variable or the `map` function would do the job.